content/2_Bronze/Intro_Sets.mdx

```diff
@@ -490,16 +496,29 @@ map<int, int> m, M;
 for (int i = 0; i < 10; ++i)
 	m[i] = i;
 
+int current_iteration = 0;
+
 for (const auto &it : m) {
-	if ([condition])
+	// only includes every third element
+
+	if (current_iteration % 3 == 0)
 		M[it.first] = it.second;
+
```

Comment:
braces
also remove the comment between the "only includes..." and the if statement since they're related